DDSaaS: ServiceNow Performance: Crawler Integration v1.0.0 (#19476)
DataDog/integrations-core: servicenow_performance/README.md

## Overview

[ServiceNow][1] is a comprehensive platform that helps streamline and automate IT service, asset, security, and configuration management processes. It enables organizations to enhance operational efficiency, mitigate risks, and improve service delivery.

> The ServiceNow integration collects below types of data:

Reviewer: updated.

1. Logs:
    * ITSM 
    * ITAM 
    * Vulnerability Response (VR) 
    * Security Incident Response (SIR) 
    * CMDB Health

2. Metrics:
    * CMDB Health 
    * SecOps Health Analytics (Vulnerability Response)

This integration collects logs and metrics from the sources listed above and sends them to Datadog for analysis with our Log and Metrics Explorer.

* [Log Explorer][2]
* [Metrics Explorer][3]

## Setup

#### Connect your ServiceNow instance to Datadog

1. Add your instance, username, password, and data preference options
    |Parameters|Description|
    |--------------------|--------------------|

>     |Instance Name|ServiceNow instance name/subdomain to connect with. [Ex. {instance-name}.service-now.com].|

Reviewer 2: ```suggestion
    |Instance Name|ServiceNow instance name/subdomain to connect with. For example, `{instance-name}.service-now.com`.|
```

Reviewer: updated

    |Username| Username of ServiceNow instance.|
    |Password| Password of ServiceNow instance.|

>     |Data Collection Type|Dropdown to select the type of data collection: preconfigured or full.|

Reviewer 2: ```suggestion
    |Data Collection Type|Use the dropdown to select the type of data collection: preconfigured or full.|
```

Reviewer: updated.

    |Get ITSM|Enable to collect ITSM data from ServiceNow. The default value is True.|
    |Get ITAM|Enable to collect ITAM data from ServiceNow. The default value is True.|
    |Get CMDB Health|Enable to collect CMDB Health data from ServiceNow. The default value is True.|

>     |Get Security Incident Response|Enable to collect Security Incident Response data from ServiceNow. The default value is False. Note: Ensure Security Incident Response Plugin is installed before enabling|

Reviewer 2: ```suggestion
    |Get Security Incident Response|Enable to collect Security Incident Response data from ServiceNow. The default value is False. Note: Ensure the Security Incident Response Plugin is installed before enabling.|
```

>     |Get Vulnerability Response|Enable to collect Vulnerability Response data from ServiceNow. The default value is False. Note: Ensure Vulnerability Response Plugin is installed before enabling|

Reviewer 2: ```suggestion
    |Get Vulnerability Response|Enable to collect Vulnerability Response data from ServiceNow. The default value is False. Note: Ensure the Vulnerability Response Plugin is installed before enabling.|
```

Reviewer: updated

2. Click the **Save** button to save your settings.

> **Note:** You can create a new user with the necessary roles in ServiceNow for the integration.

Reviewer: updated.
